src/Build.UnitTests/Evaluation/ProjectStringCache_Tests.cs

```diff
@@ -27,46 +28,53 @@ public class ProjectStringCache_Tests
         [Trait("Category", "netcore-linux-failing")]
         public void ContentIsSameAcrossInstances()
         {
-            string content = ObjectModelHelpers.CleanupFileContents(@"
+            using (TestEnvironment env = TestEnvironment.Create())
```

Comment:
Since this is already added, it's fine, but since you need this for all tests in this class it might have been easier to use class-level [shared context](https://xunit.net/docs/shared-context) to set/restore the changewave.